compose-builder/add-security.yml

```diff
@@ -154,87 +153,45 @@ services:
     restart: always
 
 # containers for reverse proxy
-  kong-db:
-    image: postgres:${POSTGRES_VERSION}
-    user: "root:root" # Note that postgres is run under the 'postgres' user, but entry point scripts need to first run as root
-    container_name: edgex-kong-db
-    hostname: edgex-kong-db
-    read_only: true
-    restart: always
-    networks:
-      - edgex-network
-    ports:
-        - "127.0.0.1:5432:5432"
-    tmpfs:
-      - /var/run
-      - /tmp
-      - /run
-    volumes:
-      - edgex-init:/edgex-init:ro,z
-      - postgres-data:/var/lib/postgresql/data:z
-      - postgres-config:/tmp/postgres-config:z
-    entrypoint: ["/edgex-init/postgres_wait_install.sh"]
-    env_file:
-      # needed for vault host/port
-      - common-security.env
-      - common-sec-stage-gate.env
-    environment:
-      POSTGRES_DB: kong
-      POSTGRES_USER: kong
-      POSTGRES_PASSWORD_FILE: "${POSTGRES_PASSWORD_FILE:-/tmp/postgres-config/.pgpassword}"
-    depends_on:
-      - security-bootstrapper
-    security_opt: 
-      - no-new-privileges:true
 
-  kong:
-    image: kong:${KONG_VERSION}
-    user: "kong:nogroup" # Note Kong already running as kong user but doesn't define a group
-    container_name: edgex-kong
-    hostname: edgex-kong
+  nginx:
+    image: nginx:alpine-slim
+    container_name: edgex-nginx
+    hostname: edgex-nginx
     read_only: true
     restart: always
     networks:
-      - edgex-network
+      edgex-network:
+        aliases:
+          - edgex-kong  # temp patch for edgex-ui-go
     ports:
-        - "8000:8000"
-        - "8443:8443"
-        - "127.0.0.1:8100:8100"
-    tty: true
-    entrypoint: ["/edgex-init/kong_wait_install.sh"]
+      - "8443:8443"  # port 8000 (insecure) is not mapped externally)
+    entrypoint: ["/bin/sh", "/edgex-init/nginx_wait_install.sh"]
+    command:
+      - "/docker-entrypoint.sh"
+      - nginx
+      - "-g"
+      - "daemon off;"
     env_file:
       - common-sec-stage-gate.env
-    environment:
-      KONG_DATABASE: postgres
-      KONG_PG_HOST: edgex-kong-db
-      KONG_PG_PASSWORD_FILE: "${KONG_POSTGRES_PASSWORD_FILE:-/tmp/postgres-config/.pgpassword}"
-      KONG_PROXY_ACCESS_LOG: /dev/stdout
-      KONG_ADMIN_ACCESS_LOG: /dev/stdout
-      KONG_PROXY_ERROR_LOG: /dev/stderr
-      KONG_ADMIN_ERROR_LOG: /dev/stderr
-      KONG_ADMIN_LISTEN: "127.0.0.1:8001, 127.0.0.1:8444 ssl"
-      KONG_STATUS_LISTEN: "0.0.0.0:8100"
-      KONG_DNS_ORDER: "LAST,A,CNAME"
-      KONG_DNS_VALID_TTL: '1'
-      KONG_SSL_CIPHER_SUITE: modern
-      KONG_NGINX_WORKER_PROCESSES: '1'
-    tmpfs:
-      - /run
-      - /tmp
     volumes:
       - edgex-init:/edgex-init:ro,z
-      - postgres-config:/tmp/postgres-config:z
-      - kong:/usr/local/kong:z
-      - /tmp/edgex/secrets/security-proxy-setup:/tmp/edgex/secrets/security-proxy-setup:ro,z
+      - nginx-templates:/etc/nginx/templates
+      - nginx-tls:/etc/ssl/nginx
+      #- nginx-conf:/etc/nginx/conf.d
```

Comment:
Why is this here commented out?

Reply:
TODO added.  I made conf.d a tmpfs mount instead.  Removed nginx-conf volume and this commented line.